contrib/babelfishpg_common/sql/geometry.sql

```diff
@@ -1,6 +1,6 @@
 CREATE OR REPLACE FUNCTION sys.geometryin(cstring)
     RETURNS sys.GEOMETRY
-    AS '$libdir/postgis-3', 'LWGEOM_in'
+    AS 'babelfishpg_common', 'LWGEOM_in'
```

Comment:
Let's re-name the C function as `geometry_in`

Reply:
Done